joern-cli/frontends/kotlin2cpg/src/main/scala/io/joern/kotlin2cpg/ast/AstForDeclarationsCreator.scala

```diff
@@ -75,7 +75,8 @@ trait AstForDeclarationsCreator(implicit withSchemaValidation: ValidationMode) {
       classDeclarations.collectAll[KtProperty].filter(_.getInitializer != null).map { decl =>
         val initializerAsts = astsForExpression(decl.getInitializer, None)
         val rhsAst =
-          if (initializerAsts.size == 1) initializerAsts.head
+          if (initializerAsts.size == 1)
```

Comment:
I think we should choose either `Ast(unknownNode(decl.getInitializer, Constants.empty))` or `Ast(unknownNode(decl, "<empty>"))` since one branch is going to be missing here. i.e., there is no point in calling `headOption` if it's directly after `initializerAsts.size == 1` since `.head != null` is guaranteed.

Reply:
ok, keeping it as is to `initializerAsts.head`